core/src/cluster_info.rs

```diff
@@ -2589,7 +2590,7 @@ mod tests {
         }));
 
         let mut i = 0;
-        while value.size() < desired_size {
+        while value.size() <= desired_size {
```

Comment:
`<=` instead of `<` got `test_split_messages_packet_size()` working again.

Reply:
Ah okay, so the extra 2 bytes made it match exactly. I think that should be fine.